src/System.Text.Utf8/tests/TypeConstraintsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection;
+using Xunit;
+
+namespace System.Text.Utf8.Tests
+{
+    public class TypeConstraintsTests
+    {
+        [Fact]
+        public void Utf8StringIsAStruct()
+        {
+            var utf8String = "anyString"u8;
```

Comment:
Should you extract this utf8string to a field and initialize it in the constructor so that you avoid this code repetition in all test cases?